src/routes/home.tsx

```diff
@@ -5,27 +5,47 @@ import Spinner from '@/components/spinner'
 import { compareEvents } from '@/utils/compare-events'
 import { useGeoLocation } from '@/utils/use-geo-location'
 import { useEvents } from '@/cache/events/use'
+import { useState } from 'preact/hooks'
+import { ProcessedEventInfo } from '@/api/event-info'
 
 const now = new Date()
-
 const Home = () => {
   const events = useEvents()
   const location = useGeoLocation()
+  const [term = '', setTerm] = useState('');
```

Comment:
```suggestion
  const [term, setTerm] = useState('');
```

You can remove the default value because you are passing an initial value to `useState`